test/server/enrichment-map/enrichment/enrichment-test.js

```diff
@@ -1,72 +1,90 @@
 const chai = require('chai');
 const expect = chai.expect;
-const {enrichment} = require('../../../server/enrichment-map/enrichment');
+const _ = require('lodash');
+const {enrichment} = require('../../../../src/server/enrichment-map/enrichment');
+
+
+// object keys are unordered
+const objectEquality = (obj1, obj2) => {
+  _.forEach(obj1, property => {
+    if (!obj2.has(property)) { return false; }
+    if (obj1.get(property) != obj2.get(property)) { return false; }
+  });
+  return true;
+};
+
 
 describe('test enrichment', function() {
+  this.timeout(500000);
```

Comment:
why is this needed

Reply:
Removing this line sometimes gives timeout error.

Reply:
ok